modules/augment/include/opencv2/augment/resize.hpp

```diff
@@ -0,0 +1,66 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_RESIZE_HPP
+#define OPENCV_AUGMENT_RESIZE_HPP
+#include <opencv2/augment/transform.hpp>
+
+namespace cv { namespace augment {
+
+class CV_EXPORTS_W Resize : public Transform
+{
+public:
+    /* @brief Constructor
+       @param size the size the images will be resized to 
+       @param interpolations vector of the interpolation techniques to be chosen at random
+    */
+    CV_WRAP Resize(Size size, std::vector<int>& interpolations);
```

Comment:
`const std::vector<int>& `

Reply:
I fixed all the mentioned issues, but this I wrote it as:
 const std::vector<int>& interpolations = std::vector<int>()

because the python binder has a problem with the other one, it's not different though.